src/parseOption.ts

```diff
@@ -266,3 +278,28 @@ export function dropFieldsForLineArea(marktype: string, itemData: ScenegraphData
     removeFields(itemData, quanKeys);
   }
 }
+
+function defaultSort(field: string) {
+  return (a, b) => {
+    if (!isNaN(a[field]) && !isNaN(b[field])) {
```

Comment:
I think it would be more clear to use `!isNumber` from `vega-util` than the negation of it.